Makefile

```diff
@@ -33,8 +33,8 @@ coverage:
 
 .PHONY: package-quick
 package-quick:
-	@cargo package --workspace --no-verify --exclude polarity-bench --exclude test-runner --exclude lsp-browser
+	@cargo package --workspace --no-verify --exclude polarity-bench --exclude test-runner --exclude polarity-lang-lsp-wasm
 
 .PHONY: package
 package:
-	@cargo package --workspace --no-verify --exclude polarity-bench --exclude test-runner --exclude lsp-browser
+	@cargo package --workspace --no-verify --exclude polarity-bench --exclude test-runner --exclude polarity-lang-lsp-wasm
```

Comment:
I guess this is my oversight from a previous PR. The `package` target should **not** use the `--no-verify` flag.